package.json

```diff
@@ -18,7 +18,7 @@
   ],
   "repository": "yeoman/generator-generator",
   "scripts": {
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
Without coverage runs faster and should be out in. CI can added the flag for itself
```suggestion
    "test": "jest",
```